File: iLoveYou.java
/*
* I love you in Java
*/
public class HelloWorld{

    public static void main(String []args){
        iLoveYou(true);
    }
    
    static void iLoveYou(boolean loveIsTrue){
        while(loveIsTrue){
            System.out.println("I love you Angie!");
            //loveIsTrue = false; this never end baby
        }
    }
     
}

File: iLoveYou.cpp
/*
* I love you in C++
*/
#include <iostream>

using namespace std;

void iLoveYou(bool);

int main()
{
    iLoveYou(true);
    return 0;
}

void iLoveYou(bool loveIsTrue){
    while(loveIsTrue){
        std::cout << "I love you Angie!" << std::endl;
        //loveIsTrue = false; this never end baby!
    }
}
